auth/README.md

```diff
@@ -1,6 +1,6 @@
 # FirebaseUI for Android — Auth
```

Comment:
I think we need to make the value proposition more compelling on this initial page.

I would propose to have here something like [LeakCanary's description page](https://github.com/square/leakcanary):
1) How does this help me? -> Big GIF or some other representation of what you get with FirebaseUI Auth
2) How much will it cost me? -> Let's put a snippet with the minimum code needed to run it. I'd say: gradle includes + a snippet with the check of whether the user is signed-in and if not, an invocation to the sign-in flow with Google and Email, that don't require any extra configuration.
3) How can I learn more? -> Pointers to advanced configuration and so on.